src/PowerShellEditorServices/Services/PowerShellContext/Session/Host/EditorServicesPSHostUserInterface.cs

```diff
@@ -312,7 +312,7 @@ public void SendControlC()
                         keyValuePair.Key,
                         keyValuePair.Value != null
                             ? PSObject.AsPSObject(keyValuePair.Value)
-                            : null);
+                            : string.Empty);
```

Comment:
Maybe the comment is wrong "Convert all values to PSObjects" but if not, maybe this should be:
```
psObjectDict.Add(
    keyValuePair.Key,
    PSObject.AsPSObject(keyValuePair.Value ?? string.Empty));
```

Reply:
yours is nicer 😄 went with that.